### PR TITLE
[legacy-dialog]: add sound

### PR DESCRIPTION
**Pull request by Artess999**

### CODE REVIEW

File: src/libs/dialog/src/legacy_dialog.cpp [hunk 1]
```diff
@@ -409,6 +431,26 @@ void LegacyDialog::DrawBackground(size_t start, size_t count)
...
+
+    const auto model = dynamic_cast<MODEL *>(core.GetEntityPointer(headModel_));
+
+    model->GetAnimation()->CopyPlayerState(0, 1);
+
+    model->GetAnimation()->Player(0).SetAction(action.c_str());
+    model->GetAnimation()->Player(0).Play();
+
+    model->GetAnimation()->Timer(0).ResetTimer();
+    model->GetAnimation()->Timer(0).Start(0.2f);
```
Comment (Hammie) [inline src/libs/dialog/src/legacy_dialog.cpp:459]: Why `0.2f`?

Comment (Artess999) [reply src/libs/dialog/src/legacy_dialog.cpp:459]: This value is taken from original code
Without it animationt don't changing

**Review by Hammie — APPROVED** [review]

**Comment by espkk** [comment]
Hold this a bit until I find out how to deal with broken sound slots

**Comment by espkk** [comment]
#413 should fix `SoundIsPlaying` issue

**Pull request merged**